content/news/031/index.md

```diff
@@ -99,6 +100,18 @@ Twitch][rust-gamedev-twitch].
 [gamedev-meetup-form]: https://forms.gle/BS1zCyZaiUFSUHxe6
 [rust-meetup-feb-time]: https://everytimezone.com/s/c1b2eb7b
 
+## Rust GameDev Showcase
+
+The Rust Gamedev Working Group is working on a showcase of games made in Rust!
+If you're interested in submitting your own footage, fill out [this
+form][showcase-form]. You can also read more about the process of creating the
+showcase in the [showcase discussion on Github][showcase-discussion]. The
+deadline for submissions has been extended until the 19th of February. Hope to
+see your game in there!
+
+[showcase-form]: https://docs.google.com/forms/d/1MpA7J9BFXAgoCJZJrxi8r4sqzhhYCzCtFo5kXGVOZj4/edit
+[showcase-discussion]: https://github.com/rust-gamedev/wg/discussions/121
+
```

Comment:
@17cupsofcoffee also if you could take a look at this, sorry for not PRing it properly :sweat_smile:

Reply:
Looks good 👍